postgres/migrations/V1.27__job_applications_table.sql

```diff
@@ -0,0 +1,11 @@
+-- Job Applications Table
+DROP TABLE IF EXISTS job_applications;
+CREATE TABLE job_applications (
+	application_id serial PRIMARY KEY,
+	full_name VARCHAR(40) NOT NULL,
+    phone_number VARCHAR(15) NOT NULL,
+	email text NOT NULL,
+	cover_letter text,
+	jobadid INTEGER,
```

Comment:
I wasn't sure whether I should also include the job title and company name on top of the id values for the jobad and the company but I felt that if we have the id we can retrieve the other values we need.

Reply:
That's correct. The way you've done this is called "normalized" and it's usually considered the best way to do things.

Reply:
That's correct. The way you've done this is called "normalized" and it's usually considered the best way to do things.